src/traits/election/static_committee.rs

```diff
@@ -95,7 +95,7 @@ where
 
     type QuorumCertificate = LEAF::QuorumCertificate;
 
-    type DACertificate = LEAF::DACertificate;
+    type DACertificate = DACertificate<TYPES>;
```

Comment:
Nit: I'm curious why this change is needed for DAC but not for QC.

Reply:
It's because the DAC is based of the blocks commitment rather than the leaf commitment and the LeafType::DAC bounds DAC with `SignedCertificate<..., Self>`.  So the bounds are mismatched with Election bound DAC with `SignedCertificate<..., TYPES::BlockType>`

I actually reverted this and updated the trait bounds in LeafType and that seems to work and make more sense